apps/biobin/src/biobin.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
```

Comment:
Add documentation

Reply:
What documentation are you referring to?